sky/clouds/service_catalog/data_fetchers/fetch_aws.py

```diff
@@ -73,7 +74,14 @@ def get_enabled_regions() -> Set[str]:
     global regions_enabled
     if regions_enabled is None:
         aws_client = aws.client('ec2', region_name='us-east-1')
-        user_cloud_regions = aws_client.describe_regions()['Regions']
+        try:
+            user_cloud_regions = aws_client.describe_regions()['Regions']
+        except aws.botocore_exceptions().ClientError:
```

Comment:
Should we parse it and match on a more specific Unauthorized code?

https://boto3.amazonaws.com/v1/documentation/api/latest/guide/error-handling.html#parsing-error-responses-and-catching-exceptions-from-aws-services

Reply:
Yes that makes sense, thanks!